src/main/sensors/gyro.h

```diff
@@ -56,6 +56,21 @@ typedef enum {
     GYRO_OVERFLOW_CHECK_ALL_AXES
 } gyroOverflowCheck_e;
 
+#if defined(USE_GYRO_IMUF9001)
+typedef enum {
+    IMUF_RATE_32K = 0,
+    IMUF_RATE_16K = 1,
+    IMUF_RATE_8K = 2,
+    IMUF_RATE_4K = 3,
+    IMUF_RATE_2K = 4,
+    IMUF_RATE_1K = 5,
+    IMUF_RATE_05K = 6,
```

Comment:
Is this appropriate naming convention given there's no decimal?  why not just `IMUF_RATE_500`,  `IMUF_RATE_250`, `IMUF_RATE_125`,  etc.

Reply:
I'm going to also keep them in sync automatically with the dropdown

Reply:
I also removed the lower denominations since it doesn't make sense to and I can constrain the value when set to be 5 (1k) or less based on the gyro denom.